internal/provider/origin_resource.go

```diff
@@ -145,6 +148,72 @@ func (r *OriginResource) Delete(ctx context.Context, req resource.DeleteRequest,
 	}
 }
 
+func (*OriginResource) ImportState(
+	ctx context.Context,
+	req resource.ImportStateRequest,
+	resp *resource.ImportStateResponse,
+) {
+	idParts := strings.Split(req.ID, ",")
+	if len(idParts) < 2 {
+		resp.Diagnostics.AddError(
+			"Invalid Import Identifier",
```

Comment:
Proc vlastne na ten import nejde pouzit jen a pouze id?

Reply:
Z API tyhle informace nevytáhneš. Takže jen z ID + API nejde vytvořit kompletní resource.

Reply:
Nechapu, jak to myslis. Co nejde vytahnout z API?

Reply:
Např. u AWS nejde získat `aws_access_key_secret`; je to něco co klient posílá nám při vytváření originu. https://client.cdn77.com/support/api-reference/v3/origin#detail-of-aws-origin

Reply:
Jasne, to mi nedoslo.